docs/source/model_design/vivarium_model_components/risk_factors/risk_correlation/index.rst

```diff
@@ -33,83 +36,185 @@ Instead of sampling from a univariate distribution for each risk exposure at ini
 static throughout a simulant's life.
-These percentiles, also known as propensities, are applied to different distributions as the simulant changes age group,
+These quantile ranks, also known as propensities, are applied to different distributions as the simulant changes age group,
 which means that a simulant's risk exposure in one age group and the next are perfectly auto-correlated.
 
 For example, a simulant might have their propensities for height and weight sampled at birth from a
-multivariate uniform distribution with correlation.
-If the simulant receives a propensity of 0.85 (85th percentile) in height and 0.80 (80th percentile) in weight,
+bivariate distribution in which the two components are correlated and are each uniformly distributed (i.e., a bivariate `copula <https://en.wikipedia.org/wiki/Copula_\(probability_theory\)>`_).
+If the simulant receives a propensity of 0.85 (85th percentile rank) in height and 0.80 (80th percentile rank) in weight,
 they are initially assigned the height and weight values at those percentiles among newborns.
 But when they age into the next age group, they are assigned the values at those same percentiles in their new
 age group, and so on throughout their life.
 In this way, their height and weight can change over time, but the correlation introduced in the initial sampling
 of propensities remains.
 
+PAFs for correlated risks
+-------------------------
+
+When we introduce correlation between two risk factors that have effects on the same outcome, we have to
+be especially careful about calculating the amount of that outcome attributable to these risks.
+
+One way to think about this is in terms of the population attributable fraction (PAF), which is defined as the proportion of
+the outcome that would disappear if everyone in the population was at the lowest-risk level of the risk
+exposure -- also known as the theoretical minimum risk exposure level (TMREL).
+We want to determine the "joint" PAF of our two correlated risk factors -- the proportion of
+the outcome that would disappear if everyone in the population was at the lowest-risk level of **both**
+risk exposures.
+
+By default, Vivarium uses GBD-calculated PAFs for each risk-outcome pair and combines them using
+a multiplicative approach (described below).
+This is correct when risks are all independent, but it has two problems in the presence of correlated risks:
+
+* The GBD approach to calculating PAFs is biased in the presence of any confounding.
+  If we are choosing to model correlated risk exposures with effects on the same outcome,
+  that means we believe there is **at least** one confounding variable in each PAF calculation.
+  If risks (with RR > 1) are positively correlated, the GBD approach will underestimate the PAF of each;
+  this issue is explained in more detail :ref:`on the PAFs page <pafs>`.
```

Comment:
This is maybe nit-picky, but it could be helpful to note that this bias is present even if the risk effects are adjusted for confounding. My first reaction when I was reading this was to say, but GBD already adjusts for that confounding so it shouldn't matter! Until I followed the link and remembered the more nuanced point.